plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergParquetConnectorTest.java

```diff
@@ -22,4 +24,22 @@ public TestIcebergParquetConnectorTest()
     {
         super(PARQUET);
     }
+
+    @Override
+    protected boolean supportsRowGroupStatistics(String typeName)
+    {
+        return !(typeName.equalsIgnoreCase("varbinary") ||
+                typeName.contains("decimal") ||
```

Comment:
would be awesome to follow up on this

Reply:
if this wasn't excluded, the PR would fully complete https://github.com/trinodb/trino/issues/6383 issue

Reply:
I think I found a bug in the Decimal parsing for Parquet statistics, I'll open it in a separate PR.